test/musica/CMakeLists.txt

```diff
@@ -75,7 +75,15 @@ add_custom_target(
 )
 
 add_test(
-  NAME test_metadata
+  NAME test_metadata_against_source_code
   COMMAND ${Python_EXECUTABLE} ${CMAKE_BINARY_DIR}/../$ENV{CCPP_FORTRAN_TOOLS_PATH}/offline_check_fortran_vs_metadata.py
   --directory ${CMAKE_BINARY_DIR}/metadata_test
+)
+
+# Test metadata against the CCPP standard names
+add_test(
+  NAME test_metadata_against_ccpp_standard_names
+  COMMAND ${Python_EXECUTABLE} ${CMAKE_BINARY_DIR}/../$ENV{CCPP_STD_NAMES_TOOLS_PATH}/meta_stdname_check.py
+  --metafile-loc ${CMAKE_BINARY_DIR}/metadata_test/musica_ccpp.meta
+  --stdname-dict ${CMAKE_CURRENT_SOURCE_DIR}/../../schemes/musica/musica_ccpp_namelist.xml
```

Comment:
Just to confirm, is this testing the MUSICA code against a custom MUSICA dictionary of standard names? schemes/musica/musica_ccpp_namelist.xml seems to include a set of name list options and not a standard name dictionary like in the CCPPStandardNames repository (https://github.com/ESCOMP/CCPPStandardNames/blob/main/standard_names.xml). Maybe I misunderstood the `meta_stdname_check.py`, so I'm just confirming this is intended. Thanks!

Reply:
That’s a good question. As far as I know, the current XML file doesn't include any MUSICA standard names yet, and we haven't discussed it so far. The current test primarily checks if the XML formatting is correct and identifies any names that are not in the Standard Name lists. I think this is something we should discuss further. Hi @mattldawson,  do you have any feedback on this test?

```
Non-dictionary standard names found in the following metadata files:
    - air_pressure
    - air_temperature
    - blackbody_temperature_at_surface
    - ccpp_constituent_properties
    - ccpp_constituents
    - ccpp_error_code
    - ccpp_error_message
    - dry_air_density
    - dynamic_constituents_for_musica_ccpp
    - extraterrestrial_radiation_flux
    - geopotential_height_wrt_surface
    - geopotential_height_wrt_surface_at_interface
    - micm_solver_type
    - number_of_grid_cells
    - number_of_photolysis_wavelength_grid_sections
    - photolysis_wavelength_grid_interfaces
    - standard_gravitational_acceleration
    - surface_albedo_due_to_UV_and_VIS_direct
    - surface_geopotential
    - timestep_for_physics
    - vertical_interface_dimension
    - vertical_layer_dimension
```

Reply:
I guess I missed that part. I think I agree with @jimmielin - we should be checking against the standard names in `ESCOMP/CCPPStandardNames` in addition to the very few MUSICA-specific standard names (in the `musica_ccpp_namelist.xml`). I think many of the names in the list above should already be in the official list (if we spelled them correctly).

Reply:
Okay, thank you both! I will rework on this.

Reply:
Here is the updated list of items that are not present in the dictionary. 

```
Non-dictionary standard names found in the following metadata files:
Found in the spreadsheet
    - blackbody_temperature_at_surface
    - extraterrestrial_radiation_flux
    - geopotential_height_wrt_surface_at_interface
    - number_of_photolysis_wavelength_grid_sections
    - photolysis_wavelength_grid_interfaces
    - surface_albedo_due_to_UV_and_VIS_direct

Listed in musica_ccpp_namelist.xml
    - micm_solver_type
    - number_of_grid_cells

Not applicable to either, but there's no need to worry about
    - dynamic_constituents_for_musica_ccpp
```